www/src/data/sidebars/doc-links.yaml

```diff
@@ -277,6 +277,8 @@
               link: /docs/building-a-site-with-authentication/
             - title: Adding Forms
               link: /docs/adding-forms/
+            - title: Building a Contact Form
```

Comment:
I think this is fine for now actually (and simpler), since there aren't any plans for other form guides at the moment. We could always make it a sub-item later on, since it doesn't impact the URL.